web_timeline/static/src/js/timeline_controller.js

```diff
@@ -226,13 +229,17 @@ var CalendarController = AbstractController.extend({
         });
     },
 
-    write_completed: function () {
+    write_completed: function (options) {
         var params = {
             domain: this.renderer.last_domains,
             context: this.context,
             groupBy: this.renderer.last_group_bys,
         };
-        this.update(params, null);
+
+        var defaults = _.defaults(options || {}, {
```

Comment:
`_.defaults({}, options, {` please. This way we avoid modifying the `options` object in place.